File: Api/SGQ.GDOL/SGQ.GDOL/SGQ.GDOL.Domain/ObraRoot/Entity/Ocorrencia.cs
using System;

namespace SGQ.GDOL.Domain.ObraRoot.Entity
{
    public class Ocorrencia : EntityBase
    {
        public int IdInspecaoObra { get; set; }
        public DateTime? DataDescricao { get; set; }
        public DateTime? DataTratativa { get; set; }
        public string Descricao { get; set; }
        public string Tratativa { get; set; }
        public bool? Delete { get; set; }
        public DateTime? DataHoraInclusao { get; set; }
        public DateTime? DataHoraAlteracao { get; set; }

        public virtual InspecaoObra InspecaoObra { get; set; }
    }
}

File: App/sgq/src/models/ocorrencia.ts
export class Ocorrencia {
    
    id: number = 0;
    idGuidOcorrencia: string = '';
    idInspecaoObra: number = 0;
    idGuidInspecao: string = '';
    idInspecaoObraItem: number;
    descricaoInspecaoObraItem: string = '';
    numeroInspecao: number;
    dataDescricao: string;
    dataTratativa: Date;
    descricao: string = '';
    tratativa: string = '';
    delete?: boolean = false;
    dataHoraInclusao: Date;
    dataHoraAlteracao: Date = new Date();
    
    constructor(values: Object = {}) {
        Object.assign(this, values);
    }

}